packages/use-wallet/src/wallets/lute.ts

```diff
@@ -67,7 +67,10 @@ export class LuteWallet extends BaseWallet {
 
   private async getGenesisId(): Promise<string> {
     const algodClient = this.getAlgodClient()
-    const genesis = await algodClient.genesis().do()
+    const genesisStr = await algodClient.genesis().do()
```

Comment:
This got way uglier--4 lines from 1?

Reply:
Yes, like everything else this method used to return `Promise<Record<string, any>>` which was parsable as JSON, but now is retuned as `Promise<string>` and so must be deserialized first.

Reply:
Unfortunately, yes 😅